packages/amazonq/.changes/next-release/Feature-db34038d-d2bf-4b6e-b4dc-ac84c0156ab5.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Subprocesses are now tracked and visible by \"Log All Active Subprocesses\" in command palette"
```

Comment:
Should we introduce it as a more generic `Show Stats` or `Report Performance` command? Even if it only has process stuff currently.